### DISCUSSION TITLE
I want to show the icon in linkedin(only) website post input but I cannot see my extension when i configure my content_scripts

### DISCUSSION THREAD

**Discussion by piyushyadav0191** [wxt-dev/wxt#992] [Q&A]
wxt.config.ts
```ts
import { defineConfig } from 'wxt';

// See https://wxt.dev/api/config.html
export default defineConfig({
  modules: ['@wxt-dev/module-react'],
  manifest: {
    permissions: ["activeTab"],
    content_scripts: [
      {
        "matches": ["https://www.linkedin.com/*"],
        //  "js": ["content.js"]
      }
    ]
  },
});

```

Can anyone tell why extension is not showing even on linkedin.com website

**Comment by aklinker1** [comment]
Don't list content scripts in your manifest. WXT looks for content script files in your `enterypoints/` directory, and generates the manifest for you.

Here's an example: https://github.com/wxt-dev/examples/blob/main/examples/inject-script/entrypoints/content.ts

And here's the docs: https://wxt.dev/guide/directory-structure/entrypoints/content-scripts.html

**Comment by piyushyadav0191** [comment]
Oh Thanks. I understand I am kinda confused using WXT because I always write extension in Normal Html, css, JS bundle one. 

How can i achieve this functionality?

![image](https://github.com/user-attachments/assets/75f2ea5e-ac0e-42a9-bb10-6d8058302ec1)

**Reply by 1natsu172** [reply]
@piyushyadav0191 Call `ui.mount();` `ui.remove();` at the timing of the `focus` event or control it with CSS display while it is mounted.

- About UI docs: https://wxt.dev/guide/key-concepts/content-script-ui.html
- Issue related to dynamic UI mounting: https://github.com/wxt-dev/wxt/issues/537
